js-api-spec/importer.test.ts

```diff
@@ -763,6 +764,16 @@ it('throws an ArgumentError when the result sourceMapUrl is missing a scheme', (
   });
 });
 
+runOnlyForImpl('browser', () => {
+  it('node package loader throws error in browser', () => {
+    expect(() =>
+      compileString('@use "pkg:foo";', {
+        importers: [new NodePackageImporter()],
+      })
```

Comment:
This should just test `new NodePackageImporter()` without passing it to `compileString()` now.

Reply:
Addressed in https://github.com/oddbird/sass-spec/commit/b18ff2ca5bb3dfbdc33928729ba3abda226cd553